### ISSUE TITLE
Multi modal itineraries (terminal integration)

### ISSUE THREAD

**Issue by tdurand** [tdurand/mapafortaleza#5]

**Comment by regismesquita** [comment]
I Tried doing this one using the old fusion tables api , and it was a drag, due to the limited set of GIS operations it would be highly inefficient to calculate it on the client side, I'll read the new fusion API and check if there are new options to do it, otherwise we should consider caching terminal - lines relations and lose our server free structure.

**Comment by tdurand** [comment]
Yep, i came to the same conclusion that it needs to be computed on server-side.
